sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_custom_documents_async.py

```diff
@@ -34,6 +34,17 @@
 import asyncio
 
 
+def format_bounding_region(bounding_regions):
+    if not bounding_regions:
+        return "N/A"
+    return ", ".join("Page #{}: {}".format(region.page_number, format_bounding_box(region.bounding_box)) for region in bounding_regions)
+
+def format_bounding_box(bounding_box):
+    if not bounding_box:
+        return "N/A"
+    return ", ".join(["[{}, {}]".format(p.x, p.y) for p in bounding_box])
```

Comment:
Are both of these something that might be worth adding to the model object itself so a user can also consume it?
As an example, in V3 of .NET we added a convenience `ToString` for `BoundingBox` https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldBoundingBox.cs#L64

Reply:
That's a nice convenience to have! Although, I'd rather wait and see if we get requests for it, rather than add it without validating that there's a large use case for it. Maybe we could see if there are users who mention this in UX studies.

Reply:
We did add it in Bounding Box because of customer feedback. At least in .NET :)